VGG16_Net.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<p>nitpick, but you should use <code>num_classes</code>  instead of 7 on line 7. FYI, I think your Adam parameters with the exception of the learning rate are the default values.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/Skin-Cancer-Classification/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>